### PR TITLE
Fix null exception when handle failed saving throw

### PR DESCRIPTION
**Pull request by dolenam317**

### CODE REVIEW

File: SolastaUnfinishedBusiness/Models/SrdAndHouseRulesContext.cs [hunk 1]
```diff
@@ -174,6 +174,11 @@ private static void FixDivineSmiteRestrictions()
         FeatureDefinitionAdditionalDamages.AdditionalDamagePaladinDivineSmite.attackModeOnly = true;
         FeatureDefinitionAdditionalDamages.AdditionalDamagePaladinDivineSmite.requiredProperty =
             RestrictedContextRequiredProperty.MeleeWeapon;
+        var feature = ConditionFeatRaiseShield.Features[0] as FeatureDefinitionAttributeModifier;
```
Comment (EnderWiggin) [inline SolastaUnfinishedBusiness/Models/SrdAndHouseRulesContext.cs:177]: why this change is here?

**Comment by ThyWoof** [comment]
took @EnderWiggin recommendation and fixed here: https://github.com/SolastaMods/SolastaUnfinishedBusiness/commit/e0fb005c6b957d8b0056852d863732581b1a3e17
